anaconda_example/README.md

```diff
@@ -0,0 +1,22 @@
+# Introduction
+This barebones example shows how to run TensorFlow GPU jobs on condor without using a Docker container. It uses Anaconda to install the necessary cuda libraries. Simply upload to a submit node and call `condor_submit gpu_job.sub` to run the example.
+
+# Supported Servers
+This example uses a whitelist and supports different environment setups depending on the specific server it is running on. Currently, the following servers are supported:
+- ahlquist0000.chtc.wisc.edu
+- ahlquist2000.chtc.wisc.edu
+- ahlquist2001.chtc.wisc.edu
+- gitter0000.chtc.wisc.edu
+- gpu2001.chtc.wisc.edu
+- gpu2000.chtc.wisc.edu
+- gzk-1.chtc.wisc.edu
+- gzk-2.chtc.wisc.edu
+- gzk-3.chtc.wisc.edu
+- gzk-4.chtc.wisc.edu
+- tekin0000.chtc.wisc.edu
+- vetsigian0000.chtc.wisc.edu
+
+New servers can be added in [gpu_job.sub](gpu_job.sub) and [environment_setup.sh](environment_setup.sh). In the future, this example could be updated to support more general GPU requirements instead of a server whitelist. 
+
+# GZK Workaround
```

Comment:
I think we might prefer to remove this altogether, if the gzk machines are really a special case. (I know it makes sense for some of your research to include them, but I think we need general/generalizable examples as templates for users to try.)